cmd/server-rlimit.go

```diff
@@ -65,7 +69,7 @@ func GetMaxCacheSize() (cacheSize uint64, err error) {
 
 	// Get total RAM.
 	var stats sys.Stats
-	if stats, err = sys.GetStats(); err != nil {
+	if stats, err = sys.GetStats(os.Getpid()); err != nil {
```

Comment:
Pid is needed to know the right cgroup rules for the given process. As each process can have a different cgroup.